routers/web/web.go

```diff
@@ -230,6 +230,9 @@ func RegisterRoutes(m *web.Route) {
 	// Routers.
 	// for health check
 	m.Get("/", Home)
+	m.Group("/sitemap.xml", func() {
+		m.Get("", HomeSitemap)
+	}, ignExploreSignIn)
```

Comment:
```suggestion
	m.Get("/sitemap.xml", ignExploreSignIn, HomeSitemap)
```
I don't see the benefit of using `Group` here.